compiler/il/symbol/OMRSymbol.hpp

```diff
@@ -432,6 +432,9 @@ class OMR_EXTENSIBLE Symbol
    void setNamedShadowSymbol();
    bool isNamedShadowSymbol();
 
+   void setImmutableField() { _flags2.set(ImmutableField); }
+   bool isImmutableField()  { return _flags.testAny(ImmutableField); }
```

Comment:
setImmutableField seems to be using _flags2 whereas isImmutableField is using _flags ?